internal/providers/service.go

```diff
@@ -57,13 +57,52 @@ type ProviderService interface {
 // from the state
 var ErrInvalidTokenIdentity = errors.New("invalid token identity")
 
+// ProvSvcGhOps is an interface for GitHub operations inside this module
+// It is used to mock GitHub operations in tests, but in order to generate
+// mocks, the interface must be exported
+type ProvSvcGhOps interface {
```

Comment:
I'm not a fan of deeply-abbreviated names.  They feel distracting, like I need to puzzle out all the missing letters.  😁  Since this is already in the "providers" module, can well call it something like `GitHubClient` (e.g. elsewhere it would be `providers.GitHubClient`)?

Reply:
renamed, along with rebase to the current master
thanks for pointing this out, I need to get my old C naming habits out of my system :-)